app/api/V2/models/usermodels.py

```diff
@@ -1,36 +1,57 @@
 """COntain User model."""
-users = []
-
+import psycopg2
+from flask import app
+from app.db_config import init_db
 
+connection = init_db()
 class User:
     """Class for user object."""
 
-    user_id = 1
-
-    def __init__(self, username=None, email=None, password=None):
+    def __init__(self, _id=None, role=None, username=None, email=None, password=None, phone=None):
         """Initialize User class."""
         self.username = username
         self.email = email
         self.password = password
-        self.id = User.user_id
+        self.id = _id
+        self.role = role
+        self.phone = phone
 
     def serialize_user(self):
         """Return tuple as user dictionary."""
         return dict(
             id=self.id,
             username=self.username,
             email=self.email,
-            password=self.password
+            role=self.role,
+            phone=self.phone
         )
 
-    def get_by_username(self, username):
+    @classmethod
+    def get_by_username(cls, username):
         """Find a user by username."""
-        for user in users:
-            if user.username == username:
-                return user
+        cur = connection.cursor()
```

Comment:
Kindly use a more descriptive variable name than `cur`.

Reply:
I have used cursorobject for the previously "cur" variable